usermods/TetrisAI_v2/usermod_v2_tetrisai.h

```diff
@@ -143,35 +148,37 @@ uint16_t mode_2DTetrisAI()
     }
 
     tetrisai_data->tetris = TetrisAIGame(gridWidth, gridHeight, nLookAhead, piecesData, numPieces);
+    tetrisai_data->tetris.state = TetrisAIGame::States::INIT;
     SEGMENT.fill(SEGCOLOR(1));
   }
 
   if (tetrisai_data->intelligence != SEGMENT.custom1)
   {
     tetrisai_data->intelligence = SEGMENT.custom1;
-    double dui = 0.2 - (0.2 * (tetrisai_data->intelligence / 255.0));
+    float dui = 0.2 - (0.2 * (tetrisai_data->intelligence / 255.0));
```

Comment:
To reduce storage requirements add `f` to floating constants.

Reply:
good finding. I fixed it and found a few more.